From 03d7f049bfc2617daa2bdb03d1f8361c2beee94d Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Mon, 13 May 2019 21:32:13 +0000
Subject: [PATCH] joyent/node-verror#68 typo in README Reviewed by: Isaac Davis
 <isaac.davis@joyent.com> Reviewed by: Dan McDonald <danmcd@joyent.com>
 Approved by: Isaac Davis <isaac.davis@joyent.com>

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index c1f0635..f1896df 100644
--- a/README.md
+++ b/README.md
@@ -481,7 +481,7 @@ it's a `VError` or not.
 
 MultiError is an Error class that represents a group of Errors.  This is used
 when you logically need to provide a single Error, but you want to preserve
-information about multiple underying Errors.  A common case is when you execute
+information about multiple underlying Errors.  A common case is when you execute
 several operations in parallel and some of them fail.
 
 MultiErrors are constructed as:
-- 
2.21.0

